### PR TITLE
HTTP rewrite matches the example text

### LINKED ISSUE #1338: HTTP Rewrite Docs have incorrect example
**What happened**:
The example at the end of https://gateway-api.sigs.k8s.io/guides/http-redirect-rewrite/ is incorrectly repeated and does not match the text above.

**What you expected to happen**:
The example to match the text.

/kind documentation

### PR DESCRIPTION
**Pull request by subi9**
**What type of PR is this?**

**What this PR does / why we need it**:
HTTP rewrite example matches with the yaml file example given
**Which issue(s) this PR fixes**:

Fixes #1338

**Does this PR introduce a user-facing change?**:

```release-note NONE

```

### CODE REVIEW

**Comment by robscott** [comment]
Thanks!

/ok-to-test
/approve

File: examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewrite.yaml [hunk 1]
```diff
@@ -12,6 +12,9 @@ spec:
       - type: URLRewrite
         urlRewrite:
           hostname: elsewhere.example
+	  path:
+            type: ReplacePrefixMatch
+	    replacePrefixMatch: /fennel
```
Comment (robscott) [inline examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewrite.yaml:17]: Although the indentation appears consistent here, it does not in the [deploy preview](https://deploy-preview-1346--kubernetes-sigs-gateway-api.netlify.app/guides/http-redirect-rewrite/). Please update this file to all use the same kind of indentation (probably spaces).

Comment (subi9) [reply examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewrite.yaml:17]: Hi @robscott , from the deploy preview it seems that the first and second Rewrite example both got changed. So in order to rectify it , I have to create a separate file and link it in the document for the second example. Please let me know.

Comment (robscott) [reply examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewrite.yaml:17]: Yep, you're right, thanks for catching that!

Comment (subi9) [reply examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewrite.yaml]: Hi @robscott , I have created a new yaml file in the same path and linked it with the md file.Please let me know if the deploy preview looks good now and also the text for both the examples.

Comment (robscott) [reply examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewrite.yaml]: Hey @subi9, thanks for the update! This looks closer but the indentation doesn't look quite right on the second example: https://deploy-preview-1346--kubernetes-sigs-gateway-api.netlify.app/guides/http-redirect-rewrite/

File: examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewritepath.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  hostnames:
+    - rewrite.example
+  rules:
+    - filters:
+      - type: URLRewrite
+        urlRewrite:
+          hostname: elsewhere.example
+	        path:
+            type: ReplacePrefixMatch
+	          replacePrefixMatch: /fennel
```
Comment (robscott) [inline examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewritepath.yaml:17]: Your indentation looks off here

**Review by robscott — COMMENTED** [review]
Thanks @subi9! Other than indentation this change LGTM. Please rebase and squash the commits into one with your next update and this should be good to go.

File: examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewritepath.yaml (hunk 2 above)
Comment (subi9) [reply examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewritepath.yaml]: Hi @robscott , Tested and fixed the indentation in the yaml file

File: examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewrite.yaml (hunk 1 above)
Comment (subi9) [reply examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewrite.yaml]: Hi @robscott ,tested and fixed the indentation in yaml file.

**Comment by robscott** [comment]
Hey @subi9, it looks like you need to be using spaces for indentation and not tabs. The actual change LGTM if you are able to fix the indentation and rebase/squash the commits.

**Comment by subi9** [comment]
HI @robscott , I have rebased and removed the tabs and fixed with proper spaces. Please review.

**Comment by robscott** [comment]
Thanks @subi9!

/lgtm
/approve

**Pull request merged**